### ISSUE TITLE
Add ModelRenderer.destroy()

### ISSUE THREAD

**Issue by bloor** [4eb0da/war3-model#7]
Don't know if v2.0.0 has this implemented, since the npm for it does not load the /renderer folder.
But for 1.2.0 which i was forced to use, the ModelRenderer class is missing a destroy method which is useful for SinglePageApplications that can't rely on "page-reload/redirect" to clear the WEBGL context.

Basically just add this:
```
    ModelRenderer.prototype.destroy = function(){
        shaderProgram = false;
    };
```

**Comment by 4eb0da** [comment]
Hello!
I don't think that it would be enough. I've implemented it in other way:
https://github.com/4eb0da/war3-model/blob/master/renderer/modelRenderer.ts#L250-L267
It was added in version `2.1.0`. Does it suits your needs?

**Comment by bloor** [comment]
For me that was the minimum requirement to bypass the issue in my VueJs2 project.
Works fine and dandy. Thankyou for the fast fix ! 
Works like a charm.
https://prnt.sc/T9UFub305R2O

**Comment by 4eb0da** [comment]
@bloor I've discovered that RibbonsController and ParticlesController still contains data after `destroy` call, so i've cleaned them as well
It was released as version `2.2.0`

**Comment by bloor** [comment]
I think I might have seen some browser weird behavior after going through multiple model-swaps.
